Common/src/main/java/com/blamejared/crafttweaker/api/util/Many.java

```diff
@@ -28,12 +28,14 @@ public Many(T data, int amount, Function<T, String> commandStringFunc) {
         this.commandStringFunc = commandStringFunc;
     }
     
-    
+    @ZenCodeType.Method
+    @ZenCodeType.Getter("data")
```

Comment:
Make sure that this getter actually works, I vaguely remember there being issues with generics and getters, but I don't remember if it applies to this case

Reply:
The getter works fine, tested it with the following:
```zenscript
val manySand = <tag:items:minecraft:sand> * 35;

println(manySand.data);
println(manySand.getData());
println(manySand.amount);
println(manySand.getAmount());
```

Which outputs:
```
<tag:items:minecraft:sand>
<tag:items:minecraft:sand>
35
35
```